code/game/objects/random/food.dm

```diff
@@ -40,7 +40,32 @@
 	spawn_nothing_percentage = 60
 
 
+/obj/random/pizza
+	name = "random pizza"
+	icon_state = "food-red"
+
+/obj/random/pizza/item_to_spawn()
```

Comment:
I have an issue where this only pulls the first subtype instead of pulling all of them randomly, other than that this is ready to merge. So only Margherita and space cola will spawn from these respectively.

Reply:
uhhh... Return statement breaks the cycle. 
You should pick from a populated list instead.
Otherwise it takes the first subtype which is Margherita, returns it and breaks the loop.